www/source/demo/windows/steps/1.html.slim

```diff
@@ -0,0 +1,42 @@
+section
+ h1 Build your ASP.NET web application with Habitat
+
+ p This guide shows how to build and run your ASP.NET web application with Habitat on Windows.
+
+ h2 In this guide
+
+ p The following steps provide a framework for how to build and package your ASP.NET application; however, you are not required to read them in order. Feel free to move back and forth between topics as you create the plan for your web application.
+
+ ul
+   li #{link_to 'Create Your Plan',"/demo/windows/steps/2"}: Learn how to create your plan.
+   li #{link_to 'Define Dependencies and Callbacks',"/demo/windows/steps/3"}: Understand how to add runtime dependencies for your web application.
+   li #{link_to 'Define Build Logic For Your App',"/demo/windows/steps/4"}: Define how source code for your web application is built and installed as part of the package creation process.
+   li #{link_to 'Add Configuration',"/demo/windows/steps/5"}: Specify which settings on your web application can be configured at runtime.
+   li #{link_to 'Leveraging Powershell DSC in Habitat Templates',"/demo/windows/steps/6"}: Use Powershell DSC to enable and configure IIS using Habitat Templates and Handlebars syntax.
+   li #{link_to 'Define Hooks',"/demo/windows/steps/7"}: Define runtime behavior for your application.
+   li #{link_to 'Build and Test',"/demo/windows/steps/8"}: Learn how to build, test, and run your web application package.
+   li #{link_to 'Connect to a Database',"/demo/windows/steps/9"}: Describes how to connect your web application to a SQL Server database inside of a Studio or a container.
```

Comment:
Super minor suggestion, but since the other points all begin with verbs, maybe just `Connect your web application...`?

Reply:
And I believe the proper spelling of this very important term (ha) is `Habitize`. 😄